### ISSUE TITLE
Encryption

### ISSUE THREAD

**Issue by howard9199** [JacobLinCool/Bang-C#157]
the encryption method is weak, the player can recognize the x when he used to have this card.

**Comment by JacobLinCool** [comment]
The encryption key of a card should be changed after every time the ownership of its has been transferred.

**Comment by howard9199** [comment]
Another method is changing the encryption key each round, and use SPN instead of XOR.

**Comment by JacobLinCool** [comment]
> Another method is changing the encryption key each round, and use SPN instead of XOR.

I don't think it is necessary to re-encrypt everything in each round, since the ownerships of most cards are not change.

And this may cause a problem because the client need a consistent "id" to track the transition state.
